pkg/sso/client_test.go

```diff
@@ -50,6 +59,30 @@ func TestClientAuthorizeURL(t *testing.T) {
 	}
 }
 
+func TestClientAuthorizeURLWithNoDomainAndProvider(t *testing.T) {
+	test := struct {
```

Comment:
Look like you don't need this structure since you are not running some other sub tests.
You can inline the options:
```go
u, err := client.GetAuthorizationURL(GetAuthorizationURLOptions{
    State: "state",
})
```